native/babel.config.js

```diff
@@ -11,6 +11,8 @@ module.exports = {
       plugins: ['transform-remove-console']
     }
   },
+  // prevent babel error - only windows (using backslash)
+  exclude: [/node_modules\\decamelize.*/],
```

Comment:
Instead of excluding one package we could include only the packages babel should transpile. This would lead to faster transpilation for the whole project

Reply:
I'm not sure whether that would work with metro. I think React Native packages do not always transpile properly. So babel is kind of required on the node_modules directory. Did you try it?

Reply:
I tried my solution and it works.
If we would replace it with "includes" instead of the "exclude" then we have to try it out. 
So it is by far more work and maybe we still would have to include many packages to the babel transpiler